core/templates/dev/head/domain/objects/NumberWithUnitsObjectFactory.js

```diff
@@ -21,15 +21,43 @@ oppia.constant('NUMBER_WITH_UNITS_PARSING_ERRORS', {
   INVALID_VALUE:
     'Please ensure that value is either a fraction or a number',
   INVALID_CURRENCY:
-    'Please enter a valid currency (e.g., $ 5 or ₹ 5)',
+    'Please enter a valid currency (e.g., $5 or Rs 5)',
+  INVALID_CURRENCY_FORMAT: 'Please write currency units at the beginning',
   INVALID_UNIT_CHARS:
     'Please ensure that unit only contains numbers, alphabets, (, ), *, ^, /, -'
 });
 
+oppia.constant('CURRENCY_UNITS', {
```

Comment:
Explain this structure in comment (with "how to add new unit" section). So that other devs can understand its usage.

Reply:
Done